css/button.css

```diff
@@ -0,0 +1,9 @@
+.btn-success {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  border: none;
+  border-radius: 4px;
+}

```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/205329410-1235aee4-aa1b-4eeb-9a83-f34fc45224c8.png)

Please make sure to use the corresponding button effects 😉:
![image](https://user-images.githubusercontent.com/42799579/205329567-57a46edc-fda3-48fb-9e9f-a6dc71166274.png)

Tip: to make the cursor point when hovering elements, you can use:
```css
cursor: pointer;
```